app/models/invenio_rdm_record_converter.rb

```diff
@@ -50,4 +53,20 @@ def versions(gf)
       return [] unless gf.content.has_versions?
       Sufia::Export::VersionGraphConverter.new(gf.content.versions).versions
     end
+
+    def invenio_metadata(gf)
```

Comment:
I was going to say just remove this method, and have the following three lines on line 20, but I think this looks much cleaner.